ansible/configure/roles/database-import/tasks/main.yml

```diff
@@ -30,13 +38,20 @@
 - name: Create new database
   command: createdb vmdb_production
 
+- name: Create role 'cloudforms'
+  command: createuser cloudforms
```

Comment:
Is this error preventing you from running the appliance or performing any testing with respect to the UI?   I haven't seen a kbase mentioning this is required.  It's more than likely an innocuous error message.